modules/mono/build_scripts/build_assemblies.py

```diff
@@ -320,6 +320,7 @@ def generate_sdk_package_versions():
 
     constants = """namespace Godot.SourceGenerators
 {{
+#pragma warning disable IDE0040 // Add accessibility modifiers. This is currently disabled because of https://github.com/dotnet/roslyn/issues/52904
```

Comment:
We should add a `TODO:` so we can find it easily later, preferably at the start of a comment.

```suggestion
#pragma warning disable IDE0040 // Add accessibility modifiers.
    // TODO: This is currently disabled because of https://github.com/dotnet/roslyn/issues/52904
```